text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Token API will provide token information by requesting `hathor-wallet-service`.
+
+# Motivation
+[motivation]: #motivation
+
+We need to retrieve token data without requesting the full node. Wallet service holds those data, so, we'll use it for now. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+- We'll basically act as a proxy making requests to wallet service
+
+# Reference-level explanation
```

Comment:
I'm wondering whether we will just relay the response from the wallet service or not. I mean, should we have a well defined return type and map the Wallet Service's response into it? Or should we just be a reverse proxy and relay it?

I feel this is important to understand how this API might be affected by future changes in the Wallet Service's Token API.